core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/STDCMNode.kt

```diff
@@ -44,6 +53,25 @@ data class STDCMNode(
         // complete the path
         return if (abs(runTimeEstimation - otherRunTimeEstimation) >= 1e-3)
             runTimeEstimation.compareTo(otherRunTimeEstimation)
+        // Minimise the difference with the planned arrival times
+        else if (
+            plannedTimingData != null &&
+                other.plannedTimingData != null &&
+                abs(
+                    (plannedTimingData.arrivalTime.seconds - time) -
+                        (other.plannedTimingData.arrivalTime.seconds - other.time)
+                ) > 1e-3
```

Comment:
That epsilon should probably become a constant (or you could create a comparison helper function, it might make those comparison more readable)

Reply:
I think I'll create a helper function. I don't want people to confuse this with the `TIME_EPSILON = 1e-2` in `TrainPhysicsIntegrator.java`, as we really want to have a precision of 1-e3 here in my opinion.